packages/cli/src/nonInteractiveCli.test.ts

```diff
@@ -231,7 +232,7 @@ describe('runNonInteractive', () => {
     expect(processExitSpy).not.toHaveBeenCalled();
     expect(mockGeminiClient.sendMessageStream).toHaveBeenCalledTimes(2);
     expect(processStdoutSpy).toHaveBeenCalledWith(
-      "Sorry, I can't find that tool.",
+      expect.stringContaining("Sorry, I can't find that tool."),
```

Comment:
nit: maybe include your whole descriptive message to make sure someone doesn't accidentally remove it?